stl/src/filesystem.cpp

```diff
@@ -546,16 +532,11 @@ _Success_(return == __std_win_error::_Success) __std_win_error
         return {false, _Translate_not_found_to_success(_Last_error)};
     }
 
-    // For Windows 10 1809 or later we have this flag -> FILE_DISPOSITION_FLAG_IGNORE_READONLY_ATTRIBUTE 0x10
-    // This flag also deletes read-only files.
-    // NOTE: This is currently undocumented in MSDN. Refer to WinBase.h for declarations.
-
-    // The following bits are set here
-    // FILE_DISPOSITION_FLAG_DELETE, FILE_DISPOSITION_FLAG_POSIX_SEMANTICS,
-    // FILE_DISPOSITION_FLAG_IGNORE_READONLY_ATTRIBUTE

```

Comment:
This still seems to be undocumented - should we keep this comment around? (https://docs.microsoft.com/en-us/windows/win32/api/minwinbase/ne-minwinbase-file_info_by_handle_class says that `FileDispositionInfoEx` exists, but refuses to say anything about it. As far as https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-setfileinformationbyhandle is concerned, it may as well not exist.)

Reply:
Per discussion with Stephan, this comment really doesn't contain much more info than `Windows 10 1809 added support for FILE_DISPOSITION_FLAG_IGNORE_READONLY_ATTRIBUTE.` conveys. It never explained what `FILE_DISPOSITION_INFO_EX` is or how it relates to `FILE_DISPOSITION_INFO`, so nevermind.